apps/bfd-data-fda/src/test/java/gov/cms/bfd/data/fda/lookup/FdaDrugCodeDisplayLookupTest.java

```diff
@@ -49,7 +55,10 @@ void setup() {
   /** Return Fake Drug Code when parameter is true. */
```

Comment:
I think this comments about the true/false parameter are no longer valid now that the parameter is gone.